MODULE.bazel

```diff
@@ -17,6 +17,7 @@ bazel_dep(
     name = "rules_python",
     version = "1.2.0",
 )
+bazel_dep(name = "rules_cc", version = "0.0.17")
```

Comment:
I wonder why we need rules_cc to build the plugin?

Reply:
Because they were referred to from the aspect. I didn't know that you used templating for that. I'll do a second PR to address JavaInfo and java_common use in the aspects. Since I'm not familiar with the setup, it would be useful if somebody else can fix @rules_cc references in the aspect to use templates.

Reply:
I had to add it back. There are references from the tests that are located in main repository. I added it as a dev_dependency, though.